Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -360,7 +360,7 @@ JSC_DEFINE_JIT_OPERATION(operationWasmToJSExitMarshalArguments, EncodedJSValue,
         case TypeKind::Externref:
         case TypeKind::Funcref:
         case TypeKind::I32: {
-            if (wasmParam.isStackArgument()) {
```

Comment:
I can't leave a comment on the line, but:
```
wasmCallingConvention().callInformationFor(typeDefinition, CallRole::Callee);
```
You could also change that to caller, right? I wonder which makes more sense here

Reply:
Agh, that's a good catch, that would probably be better.